src/views/DeleteAccount.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+    <b-form id="login-container">
+        <a class="close-button" @click="back"></a>
+        <div class="logo" />
+        
+        <p style="font-weight: bold">Are you sure you want to delete your account?</p>
+        <p>
+            We're sorry to hear that you're leaving us! Something wrong that we can help with?
```

Comment:
I would change it to "Is there something wrong that we can help with?".

Reply:
Yep, good call